README.md

```diff
@@ -4,15 +4,15 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
-### `npm start:dev`
+### `npm run start`
```

Comment:
@mick-jones Minor point, but `npm start` and `npm test` are actually sufficient here. Those names are "special" in the sense that you can omit the `run`